### ISSUE TITLE
Would it be possible to have a Maui splash screen sample?

### ISSUE THREAD

**Issue by veikkoeeva** [dotMorten/WinUIEx#66]
Hi!

I started learning a bit of Maui and got stuck with the idea of creating a splash screen worth start debugging. Alas it seem to be a bit complicated to start from that route, and so found this library of yours.

I read https://dotmorten.github.io/WinUIEx/concepts/Maui.html but I'm not sure how could I use it to create https://github.com/dotMorten/WinUIEx/blob/main/docs/concepts/Splashscreen.md on a Maui app (any page for static, video, lottie whatever example, what there is).

Would it be possible to have a sample for this? Regardless, sponsored coffee for the the benefit of the larger community. :)

**Comment by dotMorten** [comment]
It's on my list to figure out as well, but just haven't gotten that far just yet. I'll follow up here once I get a chance to wrap my head around it. What I've done so far is to just use a SplashPage in the main maui window, and then redirect to the shell page once loading is done - not quite a splashscreen but got the job done and worked the same on all platforms.
An alternative is to just hide the main window (minimize + hide from taskbar) using the window extensions in WinUIEx, then re-enable once splash screen is done showing.

Also thank you so much for the coffee! Much appreciated!

**Comment by veikkoeeva** [comment]
No problem! I also noticed I was imprecise, I was specifically going for WinUI. Well, I suppose I skip that for now and doodle around other things!

**Comment by breenbob** [comment]
I think if the SplashScreen window subclass didn't expect the Window type parameter for the subsequent Window to be launched, you could easily use the approach taken [here ](https://github.com/castorix/WinUI3_SplashScreen/blob/master/App.xaml.cs)for a Maui app WinUI splash screen...

This approach immediately show splash screen Window in the WinUI App.xaml.cs constructor, then when app fully launches it creates and active the main window in OnLaunched, which in Maui terms [happens automatically in OnLaunched](https://github.com/dotnet/maui/blob/3717ce0d385d1aadbd3b9795c203d438426aae17/src/Core/src/Platform/Windows/MauiWinUIApplication.cs#L12) anyway, so no need to even override that?

**Comment by dotMorten** [comment]
Going to close this as a duplicate of #87. Working on a simpler image-based splashscreen there, that'll make splash screens for maui very simple as well.
